apps/api/src/app/events/e2e/bulk-trigger.e2e.ts

```diff
@@ -1,13 +1,18 @@
 import { expect } from 'chai';
-import axios from 'axios';
 import { MessageRepository, NotificationRepository, NotificationTemplateEntity, SubscriberEntity } from '@novu/dal';
-import { UserSession, SubscribersService } from '@novu/testing';
+import { SubscribersService, UserSession } from '@novu/testing';
 import { ChannelTypeEnum, StepTypeEnum } from '@novu/shared';
-
-const axiosInstance = axios.create();
+import { Novu } from '@novu/api';
+// eslint-disable-next-line no-restricted-imports
+import { NovuCore } from '@novu/api/core';
+// eslint-disable-next-line no-restricted-imports
+import { triggerBulk } from '@novu/api/funcs/triggerBulk';
```

Comment:
```suggestion
import type { triggerBulk } from '@novu/api/funcs/triggerBulk';
```

Reply:
cannot do that, it's failing compilation, if importing type the method cannot be called, it's for typescript compilation and intelisense only